network/hostnetwork/packet/packettype_string.go

```diff
@@ -0,0 +1,17 @@
+// Code generated by "stringer -type=packetType"; DO NOT EDIT.
```

Comment:
where is `go:generate` specified in code? (stringer call it should be part of the build process)

Reply:
fix in packet.go